Microsoft.Packaging.Tools.Trimming/docs/trimming.md

```diff
@@ -69,14 +70,33 @@ In your project (*.csproj* file) make the following change.
 </PropertyGroup>
 ```
 
-- Specify RootPackageReference to prefer avoid *rooting* packages directly reference by the project.
+- Specify RootPackageReference to avoid *rooting* packages directly referenced by the project.
 
 ```xml
 <PropertyGroup>
   <RootPackageReference>false</RootPackageReference>
 </PropertyGroup>
 ```
 
+- Specify RootProjectReference to avoid *rooting* projects directly referenced by the project.
+
+```xml
+<PropertyGroup>
+  <RootProjectReference>false</RootProjectReference>
+</PropertyGroup>
+```
+
+
+- Specify RootProjectReference to avoid *rooting* projects directly referenced by the project.
+
+```xml
+<PropertyGroup>
+  <RootProjectReference>false</RootProjectReference>
+</PropertyGroup>
+```
```

Comment:
This section appears identically twice, following the `RootPackageReference` section.